scripts/make-prod.js

```diff
@@ -0,0 +1,22 @@
+const concurently = require('concurrently');
+
+const runUnitTest = () => {
+  concurently(
+    [
+      'ENV=dev node scripts/set-env.js',
```

Comment:
Why you put `ENV=dev` here ?
Maybe explain the existing environment variable and what they do in the readme might be nice.